packages/fxa-payments-server/README.md

```diff
@@ -6,10 +6,21 @@ This is the server that handles payments.
 
 This project uses [Storybook](https://storybook.js.org/) to show each screen without requiring a full stack.
 
-You can view the Storybook built from the most recent main at http://mozilla.github.io/fxa/fxa-payments-server/
-
```

Comment:
Is there a 'latest' `main` link we can give folks who aren't concerned about a particular revision? The previous link did this. It saved people a few clicks, and took them directly to the payments server Storybook.

Reply:
Unfortunately none that I could find. I agree that would be ideal.